modules/fedimint-mint-client/src/lib.rs

```diff
@@ -104,6 +104,7 @@ enum OOBNotesData {
         peer_apis: Vec<(PeerId, SafeUrl)>,
         federation_id: FederationId,
     },
+    ApiSecret(String),
```

Comment:
TODO: should sending OOB notes pass the secret?

Reply:
@elsirion I think invites should. The apps should probably just display a warning etc. when sharing invite-only Federation.

Secrets should ideally be long and without it the UX will suffer.

And if someone really wants to share Federation publically, can just paste both the invite and secret so no point in being too bent on trying to prevent it.

Reply:
On the OOB notes I'm a bit torn. Seems like there are some fields on OOB notes to kind-of-contain an invite.

Reply:
What would the UX look like for password changes? I imagined the password to be something easy to communicate `thereisnosecondbest`, `enjoyyourpizza`, … so when people meet they can update each other verbally. Realistically who'd spend the energy to brute-force their way into a federation? If it's a long/complicated password you'll have to go through the invite QR scanning procedure again I guess?